src/Serilog/Configuration/LoggerSinkConfiguration.cs

```diff
@@ -28,6 +28,23 @@ internal LoggerSinkConfiguration(LoggerConfiguration loggerConfiguration, Action
         _addSink = Guard.AgainstNull(addSink);
     }
 
+    /// <summary>
+    /// Write log events to the specified <see cref="ILogEventSink"/>.
+    /// </summary>
+    /// <param name="logEventSink">The sink.</param>
+    /// <param name="restrictedToMinimumLevel">The minimum level for
+    /// events passed through the sink.</param>
+    /// <seealso cref="Sink(ILogEventSink, LogEventLevel, LoggingLevelSwitch?)"/>
+    /// <returns>Configuration object allowing method chaining.</returns>
+    /// <remarks>Sink configuration methods that specify <paramref name="restrictedToMinimumLevel"/> should also specify <see cref="LoggingLevelSwitch"/>.</remarks>
+    [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Why not obsolete?

Reply:
It's impossible to stop the compiler selecting it when doing overload resolution, and thus marking it obsolete results in a spurious warning.